scripts/start-release.js

```diff
@@ -61,10 +133,15 @@ async function main() {
   child.stdout.pipe(process.stdout)
   child.stderr.pipe(process.stderr)
 
+  // Wait for the versioning prompt to show up
+  await waitForPrompt(child, 'Select a new version')
```

Comment:
I don't think this is printed for canary which is why canary was working already